connectors/migrations/20240324_slackbot_whitelist.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO
```

Comment:
This migrates the existing bot names to their workspaces.
I am not putting the customers name on this PR on purpose, but doubled check them twice.